lib/swoosh/application.ex

```diff
@@ -40,30 +34,24 @@ defmodule Swoosh.Application do
             plug: Plug.Swoosh.MailboxPreview,
             options: [port: port]
           )
-          | children
         ]
       else
-        children
+        []
       end
     end
   else
-    defp runtime_children(children, :serve_mailbox) do
+    defp mailbox_children do
       if Application.get_env(:swoosh, :serve_mailbox) do
         Logger.warning("""
         Could not start preview server.
 
         Please add :plug_cowboy to your dependencies:
-          {:plug_cowboy, ">= 1.0.0"}
-
-        And then recompile swoosh:
 
-          mix deps.get
-          mix deps.clean --build swoosh
-          mix deps.compile swoosh
```

Comment:
When a dependency is added, Elixir automatically recompiles everything that optionally depends on it. :)